### PR TITLE
Support Thai language

### PR DESCRIPTION
**Pull request by chegola**
Please consider adding Thai language.

### CODE REVIEW

File: generators/languages/templates/src/main/webapp/i18n/th/metrics.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+    "metrics": {
```
Comment (deepu105) [inline generators/languages/templates/src/main/webapp/i18n/th/metrics.json]: I think you missed this file

Comment (chegola) [inline generators/languages/templates/src/main/webapp/i18n/th/metrics.json]: @deepu105  I intended to leave it as most of them are technical term which we normally use in English.

**Comment by deepu105** [comment]
@chegola could you please add this to documentation website by doing a PR to https://github.com/jhipster/jhipster.github.io repo

**Comment by chegola** [comment]
@deepu105 It's done.

**Pull request merged**